src/python/pants/pantsd/pailgun_server.py

```diff
@@ -56,12 +58,22 @@ class PailgunHandler(PailgunHandlerBase):
 
   def _run_pants(self, sock, arguments, environment):
     """Execute a given run with a pants runner."""
+    # For the pants run, we want to log to stderr.
+    # TODO Might be worth to make contextmanagers for this?
+    Native().override_thread_logging_destination_to_just_stderr()
```

Comment:
I recommend keeping the below 3 lines of code all together, i.e. removing the whitespace, because they are closely related and so that the `TODO` comment is clear that it refers to all 3 lines.